modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/ocl/loader/data/models/view/VersionOclVo.java

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Can we move the view models to the orchestrator module? seems only the orchestrator will consume these view objects.

Reply:
Sure. It's a good idea.

Reply:
but these are also exposed in the API?
I thought they are only consumed by the GUI. Is that not true?

Reply:
Yes. they are only consumed by the GUI.

Reply:
a little tricky case then. it does not belong to OCL nor to orchestrator. Can we put it in the 'api' module?

Reply:
We can't move these into  the 'api' module, which will case  Circular dependency

Reply:
It better to rename the moule  'ocl-loader', which contains all enums, querymodel, view model. Do you have some good idea?

Reply:
may be  `types` ?

Reply:
How about 'models' ?

Reply:
yes that is also fine.

Reply:
Ok. I will rename the moule 'ocl-loader' to 'models' to put all kind data object.

Reply:
sure. we can have a sub package called `ocl` under it and we can put all types related to OCL in it